package-lock.json

```diff
@@ -1,28 +1,8182 @@
 {
```

Comment:
can you please always use `npm ci` instead of `npm i`? this way you won't override the lockfile and you'll keep the same versions